app/resources/api/v2/bait_library_resource.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # Provides a JSON API representation of BaitLibrary
+    # See: http://jsonapi-resources.com/ for JSONAPI::Resource documentation
+    class BaitLibraryResource < BaseResource
+      # Constants...
+
+      # immutable # uncomment to make the resource immutable
```

Comment:
Does it need to be mutable?

Reply:
What does mutable mean in terms of API v2? Nothing can be updated via v2 right?
I am just doing a lookup, so I guess I can set it.

Reply:
Yes, some things can be updated via API v2